app/src/main/java/org/mozilla/fenix/components/searchengine/FenixSearchEngineProvider.kt

```diff
@@ -116,40 +116,47 @@ open class FenixSearchEngineProvider(
      * @return a list of all SearchEngines that are currently active. These are the engines that
      * are readily available throughout the app.
      */
-    fun installedSearchEngines(context: Context): SearchEngineList = runBlocking {
-        val installedIdentifiers = installedSearchEngineIdentifiers(context)
-        val engineList = searchEngines.await()
-
-        engineList.copy(
-            list = engineList.list.filter {
-                installedIdentifiers.contains(it.identifier)
-            }.sortedBy { it.name.toLowerCase(Locale.getDefault()) },
-            default = engineList.default?.let {
-                if (installedIdentifiers.contains(it.identifier)) {
-                    it
-                } else {
-                    null
+    fun installedSearchEngines(context: Context): SearchEngineList =
+        runBlockingIncrement {
+            val installedIdentifiers = installedSearchEngineIdentifiers(context)
```

Comment:
nit: can we avoid breaking blame? Or is it impossible because the line is too long?